filesystem/src/main/java/org/frankframework/filesystem/FileSystemSenderWithAttachments.java

```diff
@@ -91,7 +91,7 @@ public SenderResult sendMessage(Message message, PipeLineSession session) throws
 				log.error("unable to list all attachments", e);
 				throw new SenderException(e);
 			}
-			return new SenderResult(new Message(attachments.toString()));
+			return new SenderResult(attachments.asMessage());
```

Comment:
Dit was de enige usage van XmlBuilder.toString. In dit geval kon ik gelijk de asMessage gebruiken

Reply:
Ah gelukkig maar!